src/model/snake.py

```diff
@@ -14,14 +14,18 @@ class Direction(Enum):
 class Snake:
     """Representation of the Snake entity."""
 
-    def __init__(self, position: Tuple[int, int], direction: Direction):
+    def __init__(
+        self, position: Tuple[int, int], direction: Direction, size: int
+    ) -> None:
         self.position = position
         self.direction = direction
-        self.body = self._create_body(position)
+        self.body = self._create_body(position, size)
 
-    def _create_body(self, position: Tuple[int, int]) -> List[Tuple[int, int]]:
+    def _create_body(
```

Comment:
Maybe adding a comment explaining what the initial position of the snake is can help to understand the code better. I think the snake begins in a horizontal position starting from `position` and going down in the `x`-axis, but it took me some time to understand it